pkg/operator/apis/config/types.go

```diff
@@ -60,6 +60,8 @@ type ControllerConfiguration struct {
 	NetworkPolicy NetworkPolicyControllerConfiguration
 	// VPAEvictionRequirements is the configuration for the VPAEvictionrequirements controller.
 	VPAEvictionRequirements VPAEvictionRequirementsControllerConfiguration
+	// ExtensionVirtualCluster defines the configuration of the GardenExtension controller.
+	ExtensionVirtualCluster ExtensionVirtualClusterControllerConfiguration
```

Comment:
> of the GardenExtension controller.

but the struct is called `ExtensionVirtualCluster` 😕 Can you harmonize this (this occurs in multiple places)?